sdk/storage/README.md

```diff
@@ -0,0 +1,117 @@
+
+## Requirements
+
+To call Azure services, you must first have an Azure subscription. Sign up for a [free trial](https://azure.microsoft.com/en-us/pricing/free-trial/) or use your [MSDN subscriber benefits](https://azure.microsoft.com/en-us/pricing/member-offers/msdn-benefits-details/).
+
+## Need Help?
+
+Be sure to check out the [Azure Storage Forum](https://social.msdn.microsoft.com/Forums/azure/en-US/home?forum=windowsazuredata) on MSDN if you need help, or use [StackOverflow](http://stackoverflow.com/questions/tagged/azure).
+
+## Collaborate & Contribute
+
+We gladly accept community contributions.
+
+- **Issues:** Report bugs on the [Issues page](https://github.com/Azure/azure-sdk-for-cpp/issues) in GitHub. Ideally, please add an "[Azure-Storage]" prefix to the title for easier categorizing.
+- **Forums:** Communicate with the Azure Storage development team on the [Azure Storage Forum](https://social.msdn.microsoft.com/Forums/azure/en-US/home?forum=windowsazuredata) or [StackOverflow](http://stackoverflow.com/questions/tagged/azure).
+- **Source Code Contributions:** Please follow the [contribution guidelines for Azure open source](https://azure.github.io/azure-sdk/cpp_introduction.html) for instructions about contributing to the source project.
+
+For general suggestions about Azure, use our [Azure feedback forum](http://feedback.azure.com/forums/34192--general-feedback).
+
+## Download & Install
+
+### Install Dependencies
+
+#### Windows
+
+On Windows, dependencies are managed by [Vcpkg](https://github.com/microsoft/vcpkg). You can reference the [Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) to quickly set yourself up.
+After Vcpkg is initialized and bootstrapped, you can install the dependencies:
+```BatchFile
+vcpkg.exe install libxml2:x64-windows curl:x64-windows
+```
+
+#### Unix Platforms
+
+You can use the package manager on different Unix platforms to install the dependencies. The dependencies to installs are:
+
+CMake 3.15.0 and greater.
+Libxml2.
+OpenSSL.
+Libcurl.
+
+### Build from Source
+
+First, download the repository to your local folder:
+```BatchFile
+git clone https://github.com/Azure/azure-sdk-for-cpp.git
+```
+
+#### Windows
+
+##### Use CMake to generate the solution file
+```BatchFile
+cmake . -A x64 -DCMAKE_TOOLCHAIN_FILE=<YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake -DBUILD_CURL_TRANSPORT=ON -DBUILD_TESTING=ON -DBUILD_STORAGE_SAMPLES=ON -DCMAKE_CXX_FLAGS="/MP /EHsc"
+```
+
+Then you can open the generated `az.sln` to open the solution and build the library.
+The built library will be in `<ProjectDir>\<Setup>\` respectively for Azure Core and Azure Storage. e.g. `azure_core.lib` will be in `sdk\core\azure-core\Debug` for debug setup.
```

Comment:
Is it called configuration or setup?

Reply:
It's configuration, good catch, I'll change them.